### PR TITLE
feat: Show installed mods

### PR DESCRIPTION
**Pull request by GeckoEidechse**
Simply parses `enabledmods.json`. In the future we should also opt to check individual mods and compare with the JSON file.

This is #17 but with only the essential CSS changes to make the mod list properly scrollable.

### CODE REVIEW

File: src-vue/src/views/ModsView.vue
Comment (Alystrasz) [inline src-vue/src/views/ModsView.vue:40]: It's not very cool to directly display errors to end-users.
What errors do you expect here?

**Review by Alystrasz — CHANGES_REQUESTED** [review]

File: src-vue/src/views/ModsView.vue
Comment (GeckoEidechse) [reply src-vue/src/views/ModsView.vue:40]: Error is basically whatever the backend returns. Showing the backend error message unprocessed makes sense cause this allows us to show a detailed error message depending on the error (i.e. missing `enabledmods.json` vs incorrectly structured`enabledmods.json`).


Examples:

`enabledmods.json`  doesn't exist:

image

Invalid `enabledmods.json`:

image


Of course we could (should?) adjust the message that the backend actually sends but I'd do that in a separate PR once I also look into removing all the `.expect`s and `.unwrap`s. ^^

**Comment by Alystrasz** [comment]
Got it working on Linux, LGTM.
Last question before merging, is `enabledmods.json` file required for Northstar to operate?

**Comment by GeckoEidechse** [comment]
> Last question before merging, is `enabledmods.json` file required for Northstar to operate?

Nope. It generates it, if it doesn't exist.

Using `enabledmods.json` to get the list of installed mods is also not a good idea as deleted mods are never removed from that last list but this is just a first implementation, I'm changing the source for the list of installed mods in another PR in the near future ^^

**Pull request merged**
